bundles/binding/org.openhab.binding.lcn/readme_items.txt

```diff
@@ -0,0 +1,250 @@
+﻿openHAB LCN binding: Items Configuration Guide
+Author: Tobias Jüttner
+Date: 08/14/2015
+
+LCN items are defined by binding openHAB commands to LCN commands:
+Example item with 2 bindings:
```

Comment:
we must be careful with the term "binding". In openHAB context a binding is the whole bundle merely like a driver. For what is meant here we don't really have a name in openHAB. It's the "binding configuration" for this item containing one or many "Binding configuration elements".